src/server/utils/logging/config/otel.js

```diff
@@ -44,6 +45,13 @@ const { buildVersion: version } = readJsonFile('../../../.build-meta.json');
 //   });
 // }
 
+// setGlobalErrorHandler((err) => {
+//   process.stderr.write('OTel Error one-app: ');
+//   process.stderr.write(err);
+//   loggingErrorHandler(err);
+//   process.exit(1);
+// });
```

Comment:
Can we remove this instead of leaving it commented

Reply:
Dang, committed this on accident 🤦🏻